azure-pipelines.yml

```diff
@@ -600,21 +600,15 @@ jobs:
     vmImage: 'ubuntu-20.04'
   strategy:
     matrix:
-      1.0:
-        image: cblmariner.azurecr.io/base/core
-        tag: 1.0
       2.0:
-        image: mcr.microsoft.com/cbl-mariner/base/core
-        tag: 2.0
+        image: mcr.microsoft.com/cbl-mariner/base/core:2.0
+        artifact: rpm-mariner2.0
   steps:
   - task: Bash@3
     displayName: 'Build Rpm Package: Mariner'
     inputs:
       targetType: 'filePath'
       filePath: scripts/release/rpm/pipeline_mariner.sh
-    env:
-      IMAGE: $(image)
-      TAG: $(tag)
```

Comment:
https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#environment-variables

> System and user-defined variables also get injected as environment variables for your platform. When variables are turned into environment variables, variable names become uppercase, and periods turn into underscores. For example, the variable name `any.variable` becomes the variable name `$ANY_VARIABLE`.

So we don't need to explicitly declare env vars.